ceph.spec.in

```diff
@@ -159,7 +164,15 @@ BuildRequires:	gperftools-devel >= 2.6.1
 BuildRequires:	gperftools-devel >= 2.4
 %endif
 %endif
+%if 0%{?qa_run}
```

Comment:
@tchaikov What is this?  If I'm a developer I might just build the tree then executer run-standalone.sh.

Reply:
@tchaikov I think we could consider run-standalone.sh as an extension of make check.  Therefore, let's make this for make check not just for qa.

Reply:
> If I'm a developer I might just build the tree then executer run-standalone.sh.

@dzafman i think you are talking about the 3rd use case described in https://tracker.ceph.com/issues/37584#note-15. in that case, `qa_run` will be enabled by `install-deps.sh` when it found that it is working in an interactive terminal.

> I think we could consider run-standalone.sh as an extension of make check.

we could. but why? the intention of this change is to install the minimal set of "BuildRequires" dependencies.